src/main/java/com/fishercoder/solutions/_976.java

```diff
@@ -0,0 +1,21 @@
+package com.fishercoder.solutions;
+
+import java.util.Arrays;
+
```

Comment:
Could you add problem descriptions as comments in each problem as well? That'll save time for future reference.

Thanks!

Reply:
Done

Reply:
Nit: Problem title with problem # also? :)

Reply:
Added

Reply:
👍 Thanks!